lib/commands/update-kendoui.ts

```diff
@@ -62,13 +80,50 @@ class UpdateKendoUICommand implements ICommand {
 				return;
 			}
 
-			var packageIdx = parseInt(choice.packageIdx, 10)-1;
-			var downloadUri = packages[packageIdx].DownloadUrl;
 			this.updateKendoFiles(downloadUri).wait();
 
 		}).future<void>()();
 	}
 
+	private selectKendoVersion(packages: Server.IKendoDownloadablePackageData[]): IFuture<string> {
+		return ((): string => {
+			this.showTableWithAvailableVersions(packages);
+			var schema: IPromptSchema = {
+				type: "input",
+				name: "packageIdx",
+				message: "Enter the index of the package that you want to install",
+				validate: (value: string) => {
+					var num = parseInt(value, 10);
+					return !isNaN(num) && num >= 1 && num <= packages.length ? true : "Valid values are between 1 and " + packages.length;
+				}
+			};
+
+			var choice = this.$prompter.get([schema]).wait();
+			var packageIdx = parseInt(choice.packageIdx, 10) - 1;
+			var selectedPackage = packages[packageIdx];
+			this.$logger.trace("Selected package is:");
```

Comment:
Nope, I want to print the full object, in case I concatenate it with this string, I'll not receive the same output.